src/feeds/FeedReader.ts

```diff
@@ -210,7 +210,7 @@ export class FeedReader {
                     const entry = {
                         feed: {
                             title: feed.title ? stripHtml(feed.title) : null,
-                            url: url.toString()
```

Comment:
TIL strings have `toString` :thinking: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toString

Reply:
Yeah, I suspect previous versions of this code used `URL` which is why this lingered.